### ISSUE TITLE
@mentions

### ISSUE THREAD

**Issue by Jack-Papel** [mafia-rust/mafia#305]
### Backend
The backend shouldn't have to deal with @ mentions at all, as they are just a convenience for the user that will get converted before sent to the backend.
### Frontend
- [x] Determine a definite 'full name' scheme. For the rest of this issue I will write as if it were "(1) Player Name"
- [x] Messages sent containing "@1" or "@Player Name" should be transformed into "(1) Player Name"
  Unfortunately, this can't use the existing text replacement tech (#303), since it needs to happen before the message is sent.
- [x] [Won't do] ~Add autocomplete (case insensitive) for @ mentions. e.g. When typing "@ja", a tab-complete menu should appear with the options "@Jack", "@jason", etc.~
- [x] "(1)", "(2)", "(3)", etc. should be styled to stand out. Maybe with a circle background. Discuss the styling
  - [x] Decide whether all numbers should be styled, or just 1 up to the number of people in the game
- [x] Just "Player Name" should be highlighted (the way "1-Player Name") is highlighted today, with bold italics.
- [x] Whispering command should allow the following forms: "/w (1) Player Name", "/w @1" or "/w @Player Name". (Convert @'s before parsing the whisper message)
